speech/unit_tests/test__gax.py

```diff
@@ -78,3 +78,63 @@ def test_ctor(self):
         self.assertEqual(config.max_alternatives, max_alternatives)
         self.assertTrue(config.profanity_filter)
         self.assertEqual(config.speech_context.phrases, self.HINTS)
+
+
+class TestSpeechGAXMakeRequestsStream(unittest.TestCase):
+    SAMPLE_RATE = 16000
+    HINTS = ['hi']
+    AUDIO_CONTENT = b'/9j/4QNURXhpZgAASUkq'
+
+    def _callFUT(self, sample, language_code, max_alternatives,
+                 profanity_filter, speech_context, single_utterance,
+                 interim_results):
+        from google.cloud.speech._gax import _stream_requests
+        return _stream_requests(sample=sample,
+                                language_code=language_code,
+                                max_alternatives=max_alternatives,
+                                profanity_filter=profanity_filter,
+                                speech_context=speech_context,
+                                single_utterance=single_utterance,
+                                interim_results=interim_results)
+
+    def test_stream_requests(self):
+        from io import BytesIO
+        from google.cloud import speech
+        from google.cloud.speech.sample import Sample
+        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
+            SpeechContext)
+        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
+            StreamingRecognitionConfig)
+        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
+            StreamingRecognizeRequest)
+
+        sample = Sample(content=BytesIO(self.AUDIO_CONTENT),
+                        encoding=speech.Encoding.FLAC,
+                        sample_rate=self.SAMPLE_RATE)
+        language_code = 'US-en'
+        max_alternatives = 2
+        profanity_filter = True
+        speech_context = SpeechContext(phrases=self.HINTS)
+        single_utterance = True
+        interim_results = False
+        streaming_requests = self._callFUT(sample, language_code,
+                                           max_alternatives, profanity_filter,
+                                           speech_context, single_utterance,
+                                           interim_results)
+        all_requests = []
+        for streaming_request in streaming_requests:
+            self.assertIsInstance(streaming_request, StreamingRecognizeRequest)
+            all_requests.append(streaming_request)
```

Comment:
No need for a change, just a note here. You can consume an iterator by "casting" to a list or other container type, e.g.:

```python
all_requests = list(streaming_requests)
```